NEWS.md

```diff
@@ -1,5 +1,7 @@
 # dbplyr (development version)
 
+* `*_join()` now allows providing a `relationship` argument (@bairdj, #1305).
```

Comment:
Maybe be a bit more specific what is supported:

```suggestion
* `*_join()` now allows using specifying the relationship argument. It must be `NULL` or `"many-to-many"` (@bairdj, #1305).
```

Reply:
Sounds good